client/src/main/java/io/pravega/client/stream/impl/ControllerImplConfig.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import java.io.Serializable;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ControllerImplConfig implements Serializable {
```

Comment:
As far as I can tell, this configuration is supposed to be used internally when creating a controller instance, but I'm wondering if I'm missing anything.